interpreter.asm

```diff
@@ -83,7 +114,7 @@ quit_reset
 
     +BACKLINK "quit", 4
 QUIT
-    jsr quit_reset
+    jsr keep_nmi
```

Comment:
What is the purpose of not resetting $316/$318 at QUIT?

I don't mind this change but also curious why it was done :-)

Reply:
I was under the impression that QUIT was called as a matter of course with ';' I have since resolved that Idea, and I really can't see any reason this still needs to be done.